src/main/java/uk/gov/hmcts/probate/service/ConfirmationResponseService.java

```diff
@@ -275,7 +275,6 @@ private TemplateResponse generateNextStepsBodyMarkdown(CCDData ccdData) {
         keyValue.put("{{ihtForm}}", getIhtForm(ccdData));
         keyValue.put("{{additionalInfo}}", additionalInfo);
         keyValue.put("{{renouncingExecutors}}", getRenouncingExecutors(ccdData.getExecutors()));
-        keyValue.put("{{deadExecutors}}", getDeadExecutors(ccdData.getExecutors()));
```

Comment:
deadExecutors key was used only for deathAfter and deathBefore so removed from markdown